build/release-tars/BUILD.bazel

```diff
@@ -149,14 +149,14 @@ pkg_tar(
 ) for arch in archs] for os, archs in UBI_PLATFORMS.items()]
 
 [[pkg_tar(
-    name = "cert-manager-ctl-%s-%s" % (os, arch),
+    name = "kubectl-cert-manager-%s-%s" % (os, arch),
     srcs = [
         ":license-targets",
         "//:version",
```

Comment:
Do we want to include the `version` file in the inner tar.gz? I think this should only be in the outer one